src/libnnstreamer-edge/nnstreamer-edge-aitt.c

```diff
@@ -74,20 +74,11 @@ nns_edge_aitt_connect (const char *id, const char *topic, const char *host,
     nns_edge_loge ("Invalid param, handle should not be null.");
     return NNS_EDGE_ERROR_INVALID_PARAMETER;
   }
+  ah = (nns_edge_aitt_handle_s *) handle;
 
   nns_edge_logd ("Create AITT instance: broker address: %s:%d", host, port);
 
-  ah = (nns_edge_aitt_handle_s *) calloc (1, sizeof (nns_edge_aitt_handle_s));
-  if (!ah) {
-    nns_edge_loge ("Failed to allocate memory for AITT handle.");
-    return NNS_EDGE_ERROR_OUT_OF_MEMORY;
-  }
-
-  option = aitt_option_new ();
-  aitt_option_set (option, AITT_OPT_MY_IP, "localhost");
-
-  ah->aitt_handle = aitt_new (id, option);
-  aitt_option_destroy (option);
+  ah->aitt_handle = aitt_new (id, ah->option);
```

Comment:
suggestion if separating create() and connect() function,
code style - function proto
```
int nns_edge_aitt_connect (nns_edge_aitt_h handle, const char *id, const char *topic, const char *host, const int port);
```

Reply:
Agree, Updated.